Makefile

```diff
@@ -187,6 +187,16 @@ endif
 .PHONY: podman
 podman: bin/podman
 
+.PHONY: bin/podman-remote-static
+podman-remote-static: bin/podman-remote-static
+	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f Containerfile.static -t podmanbuildimage .
+	${CONTAINER_RUNTIME} run --rm --security-opt label=disable -v $$(pwd):/src/github.com/containers/libpod \
+		podmanbuildimage make podman-remote-static-local
```

Comment:
This is strange, I don't understand the purpose of this Makefile target.  It seems to build a container image, changes it's contents at runtime, then leave it on the developers workstation dangling, orphaned, and sad :cry: 

More likely: I don't understand what you're trying to accomplish here :confused:   Maybe put what I'm missing into a comment :grinning:

Reply:
You can't do this on a Fedora/RHEL/Centos box, because glibc does not support static linking.  Because of this we need to use debian or ubuntu to compile the program staticly.

This code was mostly copied from skopeo, which already went through this.

So the code creates an image and then executes a container on it, that does the build.

If you are on Ubuntu or Debian you could just do the 
make podman-remote-static-local, and save the container part.